crates/eframe/src/web/web_runner.rs

```diff
@@ -28,8 +25,10 @@ pub struct WebRunner {
     /// the panic handler, since they aren't `Send`.
     events_to_unsubscribe: Rc<RefCell<Vec<EventToUnsubscribe>>>,
 
-    /// Used in `destroy` to cancel a pending frame.
-    request_animation_frame_id: Cell<Option<i32>>,
```

Comment:
What is the relevance of the changes to the `request_animation_frame` stuff here? What was wrong with the old code?

Reply:
The official `requestAnimationFrame` example in wasm bindgen [does the same thing](https://rustwasm.github.io/wasm-bindgen/api/wasm_bindgen/closure/struct.Closure.html#using-fnonce-and-closureonce-with-requestanimationframe). The way I understand it, calling `Closure::forget()` in this case is just unnecessary GC pressure. Unlikely to matter but it was bothering me 😅.

Reply:
Could it be moved to a separate PR?

Reply:
https://github.com/emilk/egui/pull/4551